README.md

```diff
@@ -1,20 +1,20 @@
-
 # safespace
 
 <!-- badges: start -->
 <!-- badges: end -->
 
-The goal of safespace is to provide a safe space for new developers to try out a pull request (particularly fo [Hacktoberfest 2024](https://hacktoberfest.com/participation/)) 
+The goal of the `safespace` package is to provide a safe space for new developers to try out a pull request (particularly fo [Hacktoberfest 2024](https://hacktoberfest.com/participation/)) 
 for an R package where they will definitely not break any in-production code.
 
 ## Installation
 
-You can install the development version of safespace like so:
+You can install the development version of `safespace` like so:
 
 ``` r
 # istall.packages(remotes)
-remotes:install_github(jonocarroll/safespace)
+remotes::install_github("jonocarroll/safespace")
 ```
+- *(At the moment this will fail, though, because of a problem with dependency `archivedpkg`)*
```

Comment:
This is a time-sensitive note - once the issue is resolved it will no longer apply. In the interest of testing out pushing additional commits, could you please remove this line?

Reply:
Sure, it makes sense